src/ceph.in

```diff
@@ -913,17 +921,14 @@ def main():
         print(str(e), file=sys.stderr)
         return 1
 
-    if parsed_args.help:
```

Comment:
@Liuchang0812 sorry for the confusion, i mean, we should not include this change in the pep8 commit.